pages/linux/bully.md

```diff
@@ -0,0 +1,13 @@
+# bully
+
+> Brute-force the WPS pin of a wireless access point.
+> Necessary information must be gathered with `airmon-ng` and `airodump-ng` before using `bully`.
+> More information: <https://salsa.debian.org/pkg-security-team/bully>.
+
+- Crack the password:
+
+`bully --bssid {{mac}} --channel {{channel}} --bruteforce {{interface}}`
```

Comment:
```suggestion
- Crack the password:

`bully --bssid "{{mac}}" --channel "{{channel}}" --bruteforce "{{interface}}"`
```

- Optional fix: `a` may sound better.
- Optional fix: possible unsafe strings: `{{mac}}`, `{{channel}}`, `{{interface}}` can be quoted if they can contain some special characters to POSIX-like compatible shell.

Reply:
* I included `the` because MAC address obtained from previous steps is specific. What do you think? 🤔

Reply:
As a reviewer I don't see previous steps (which are previous code examples in my interpretation). If you think that `the` is more correct here - just leave it as it is. That's why I've used optional suggestion and didn't mandate you to fix this code description.

You can partially apply my suggestion: at the first step use it completely and then revert `the` -> `a` change.

Or I can just change my suggestion. :)